source/documentation/services/airflow/index.md

```diff
@@ -6,11 +6,15 @@
 
 ## Overview
 
-[Apache Airflow](https://airflow.apache.org/) is a workflow management platform for data engineering pipelines.
+[Apache Airflow](https://airflow.apache.org/) is a workflow management platform. Analytical Platform users primarily use it for:
+
+* automating data engineering pipelines
+* training machine learning models
+* reproducible analytical pipelines ([RAP](https://analysisfunction.civilservice.gov.uk/support/reproducible-analytical-pipelines/))
 
 We recommend using it for long-running or compute intensive tasks. 
 
-Pipelines are executed on the Analytical Platform's Kubernetes infrastructure and can interact with services such as Amazon Athena, Amazon Bedrock, and Amazon S3.
+Workflows are executed on the Analytical Platform's Kubernetes infrastructure and can interact with services such as Amazon Athena, Amazon Bedrock, and Amazon S3.
```

Comment:
Refactoring it a little so it refers to workflows rather than pipelines, but this is just semantics